src/runmode-unix-socket.c

```diff
@@ -367,6 +372,11 @@ static TmEcode UnixSocketAddPcapFile(json_t *cmd, json_t* answer, void *data)
         continuous = json_is_true(cont_arg);
     }
 
+    json_t *delete_arg = json_object_get(cmd, "delete-when-done");
```

Comment:
So the option is available in the unix socket running mode so this should be reflected in the documentation. Also would be good to have the option accessible in suricatasc.